instrumentation/cassandra/cassandra-4.4/testing/src/main/java/io/opentelemetry/testing/cassandra/v4_4/AbstractCassandra44Test.java

```diff
@@ -135,4 +144,22 @@ private static Stream<Arguments> provideReactiveParameters() {
                     "SELECT",
                     "users"))));
   }
+
+  @Override
+  protected CqlSession getSession(String keyspace) {
+    DriverConfigLoader configLoader =
+        DriverConfigLoader.programmaticBuilder()
+            .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(0))
+            .withDuration(DefaultDriverOption.CONNECTION_INIT_QUERY_TIMEOUT, Duration.ofSeconds(10))
+            .build();
+
+    InetSocketAddress address = new InetSocketAddress("localhost", cassandraPort);
+    return wrap(
+        CqlSession.builder()
+            .addContactEndPoint(new SniEndPoint(address, "localhost"))
```

Comment:
does this matter? if not, maybe worth using something other than "localhost" to show that it doesn't matter
```suggestion
            .addContactEndPoint(new SniEndPoint(address, "dummy"))
```